examples/panel-basic/README.md

```diff
@@ -1,22 +1,26 @@
-# Grafana Panel Plugin Template
+# Grafana Panel Plugin
 
-This is an example on how to build a basic panel plugin.
+This example provides a template for how to build a basic panel plugin. The entry point is `/src/components/SimplePanel/SimplePanel.tsx`.
 
-## What is Grafana Panel Plugin?
+By default, the plugin uses `TimeSeries` from the `grafana-ui` package to build a graph with the properties of `SimplePanel`. The plugin also allows a tooltip to be shown when user hovers over a visualization.
+
+Additionally, the plugin is configured to allow features such as a gradient mode selector and a list display mode on the Grafana sidebar.
+
+## What is a Grafana panel plugin?
 
 Panel plugins allow you to add new types of visualizations to your dashboard, such as maps, clocks, pie charts, lists, and more.
 
 Use panel plugins when you want to do things like visualize data returned by data source queries, navigate between dashboards, or control external systems (such as smart home devices).
 
 ## Getting started
 
-1. Install dependencies
+1. Install dependencies:
 
    ```bash
    yarn install
```

Comment:
not part of your PR but this bit is wrong in all the examples. this should be `npm install`. we are not using yarn in any of the examples anymore.

Reply:
I created a separate issue for this https://github.com/grafana/grafana-plugin-examples/issues/237

Reply:
I've fixed it in this PR.